snippets/csharp/VS_Snippets_CLR/CatchException/CS/catchexception2.cs

```diff
@@ -10,7 +10,7 @@ public static void Main()
         {
             StreamReader sr = File.OpenText("data.txt");
             Console.WriteLine("The first line of this file is {0}", sr.ReadLine());
```

Comment:
```suggestion
             Console.WriteLine($"The first line of this file is {sr.ReadLine()}");
```
Where code is revised, we've been replacing composite format strings with interpolated strings.

Reply:
Changed.